src/includes/DEPRECIATED-pageforms.php

```diff
@@ -0,0 +1,2 @@
+Depreciated 
```

Comment:
You can just delete this file. That will remove it from this revision. We can always go back to an earlier commit if we need the file.